run-benchmarks.sh

```diff
@@ -217,6 +240,12 @@ EXTRA_JSON=$(cat << EOF
 "empty_project_startup_shutdown_peak_memory_usage": {
 	"debug": $PEAK_MEMORY_STARTUP_SHUTDOWN_DEBUG,
 	"release": $PEAK_MEMORY_STARTUP_SHUTDOWN_RELEASE
+},
+"empty_project_editor_startup_shader_cache": {
+	"debug": $TIME_TO_STARTUP_SHADER_CACHE,
+},
+"empty_project_editor_startup_no_shader_cache": {
+	"debug": $TIME_TO_STARTUP_NO_SHADER_CACHE,
```

Comment:
JSON forbids trailing commas:

```suggestion
"empty_project_editor_startup_shader_cache": {
	"debug": $TIME_TO_STARTUP_SHADER_CACHE
},
"empty_project_editor_startup_no_shader_cache": {
	"debug": $TIME_TO_STARTUP_NO_SHADER_CACHE
```